src/Components/Pages/TabsPage.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import { Link, useParams } from 'react-router-dom';
+import classnames from 'classnames';
+
+export const TabsPage = ({ tabs }) => {
+    const { tabId } = useParams();
+
+    const currentTab = useMemo(() => (
+        tabs.find(tab => tab.id === tabId)
```

Comment:
Instead of this function you could use for conditional rendering of tab content something like
`{tabId === tab.id && tab.content}`